x-pack/plugins/fleet/README.md

```diff
@@ -54,41 +54,55 @@ It can be useful to run Fleet Server in a container on your local machine in ord
 
 _The following is adapted from the Fleet Server [README](https://github.com/elastic/fleet-server#running-elastic-agent-with-fleet-server-in-container)_
 
-1. Add the following configuration to your `kibana.dev.yml`
+1. Bootstrap Kibana
 
-```yml
-server.host: 0.0.0.0
+```
+  cd kibana
+  yarn kbn bootstrap
 ```
 
-2. Append the following option to the command you use to start Elasticsearch
 
-```
--E http.host=0.0.0.0
+2. Add the following configuration to your `config/kibana.dev.yml`
+
+```yml
+server.host: 0.0.0.0
 ```
 
-This command should look something like this:
+3. Append `-E http.host=0.0.0.0` to the command you use to start Elasticsearch. The `-E path.data` option ensures that the Elasticsearch data is persisted locally, using on the specified path:
 
 ```
+nvm use
 yarn es snapshot --license trial -E xpack.security.authc.api_key.enabled=true -E path.data=/tmp/es-data -E http.host=0.0.0.0
 ```
 
-3. Run the Fleet Server Docker container. Make sure you include a `BASE-PATH` value if your local Kibana instance is using one. `YOUR-IP` should correspond to the IP address used by your Docker network to represent the host. For Windows and Mac machines, this should be `192.168.65.2`. If you're not sure what this IP should be, run the following to look it up:
+4. On another terminal start Kibana with no base path:
 
 ```
-docker run -it --rm alpine nslookup host.docker.internal
+nvm use
+yarn start --no-base-path
```

Comment:
I am running kibana as `yarn start` only, you can configure base path in kibana.dev.yml like this:
`server.basePath: /xyz`
I think there was some problem with agent connection when I tried to run without base path.